bin/play

```diff
@@ -6,16 +6,18 @@ $: << File.join(File.dirname(__FILE__), "..", "lib")
 
 require "display"
 require "board"
-require "players"
+require "player_module"
 require "game"
 
 display = Display.new
 board = Board.new(["1", "2", "3", "4", "5", "6", "7", "8", "9"])
 display.show_welcome_instructions
 type = display.get_game_type
-players = Players.new(type, board, display)
-players.create_players
 
-game = Game.new(board, display, players.player_1, players.player_2)
+players = Player.generate(type, board)
+player_1 = players[0]
+player_2 = players[1]
+
+game = Game.new(board, display, player_1, player_2)
```

Comment:
have you used ruby's splat (*) operator?  Would mean you didn't have to do the destructuring of players-> player_1 and player_2 yourself.

Reply:
I've not used it before no. Thanks for this, I'll read up and implement it